x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/mobile/mobile_location_stats.spec.ts

```diff
@@ -230,111 +231,113 @@ export default function ApiTest({ getService }: FtrProviderContext) {
+        it('returns location for most crashes', () => {
+          const { location } = response.currentPeriod.mostCrashes;
+          expect(location).to.be('China');
+        });
 
-        expect(response.currentPeriod.mostSessions.timeseries.every((item) => item.y === 0)).to.eql(
-          true
-        );
-        expect(response.currentPeriod.mostRequests.timeseries.every((item) => item.y === 0)).to.eql(
-          true
-        );
-        expect(response.currentPeriod.mostCrashes.timeseries.every((item) => item.y === 0)).to.eql(
-          true
-        );
-        expect(response.currentPeriod.mostLaunches.timeseries.every((item) => item.y === 0)).to.eql(
-          true
-        );
+        it('returns location for most launches', () => {
+          const { location } = response.currentPeriod.mostLaunches;
+          expect(location).to.be('China');
+        });
       });
 
-      it('returns the correct values when single filter is applied', async () => {
-        const response = await getMobileLocationStats({
-          serviceName: 'synth-android',
-          environment: 'production',
-          kuery: `service.version:"1.1"`,
+      describe('when filters are applied', () => {
+        it('returns empty state for filters with no results', async () => {
+          const response = await getMobileLocationStats({
+            serviceName: 'synth-android',
+            environment: 'production',
+            kuery: `app.version:"none"`,
+          });
+
+          expect(response.currentPeriod.mostSessions.value).to.eql(0);
+          expect(response.currentPeriod.mostRequests.value).to.eql(0);
+          expect(response.currentPeriod.mostCrashes.value).to.eql(0);
+          expect(response.currentPeriod.mostLaunches.value).to.eql(0);
+
+          expect(
+            response.currentPeriod.mostSessions.timeseries.every((item) => item.y === 0)
+          ).to.eql(true);
+          expect(
+            response.currentPeriod.mostRequests.timeseries.every((item) => item.y === 0)
+          ).to.eql(true);
+          expect(
+            response.currentPeriod.mostCrashes.timeseries.every((item) => item.y === 0)
+          ).to.eql(true);
+          expect(
+            response.currentPeriod.mostLaunches.timeseries.every((item) => item.y === 0)
+          ).to.eql(true);
         });
 
-        expect(response.currentPeriod.mostSessions.timeseries[0].y).to.eql(1);
-        expect(response.currentPeriod.mostCrashes.timeseries[0].y).to.eql(1);
-        expect(response.currentPeriod.mostRequests.timeseries[0].y).to.eql(1);
-        expect(response.currentPeriod.mostLaunches.timeseries[0].y).to.eql(1);
+        it('returns the correct values when single filter is applied', async () => {
+          const response = await getMobileLocationStats({
+            serviceName: 'synth-android',
+            environment: 'production',
+            kuery: `service.version:"1.1"`,
```

Comment:
is this configuration acceptable to run tests against ESS/MKI?

Reply:
I notice tests use different query arguments and wonder if these tests will work only in KIbana CI, but when we run it against serverless on MKI and 9.0.0-snapshot ESS

Reply:
what do you mean?

Reply:
we run all deployment-agnostic tests in Cloud both for serverless (MKI) and stateful (ESS 9.0.0-snapshot). I saw you marked `MKI run` checkbox, so my guess all mobile tests from PR are check against MKI. Do we expect the same for ESS Cloud?